docs/guides/query-kyc-state-on-c-chain.md

```diff
@@ -0,0 +1,164 @@
+---
+sidebar_position: 45
+title: Query KYC State on C-Chain
+description: This document describes how to use Remix IDE to query KYC State of an address on C-Chain
+---
+
+# How to Query KYC State on C-Chain
+
+This document provides a guide on how to use Remix IDE to query the KYC state of
```

Comment:
might be useful to add the link to remix: https://remix.ethereum.org/

Reply:
We already have the link down below where it's needed. But I will add it into the this part as well.